code-scanning/codegurureviewer.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+#
+# For more information on Amazon CodeGuru Reviewer in general, see
+# https://github.com/aws-actions/codeguru-reviewer
+
+name: CodeGuru Reviewer
+
+on:
+  push:
+    branches: [ $default-branch, $protected-branches ]
+  pull_request:
+    branches: [ $default-branch ]
+  schedule:
+    - cron:  $cron-weekly
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+  steps:
+    # Step 1: Checkout the repository and provide your AWS credentials
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
```

Comment:
I'm afraid we need a full SHA here, rather than a tag.

Reply:
This step is to use a GitHub action, same as the usage of `uses: actions/checkout@v2`. 
I don't think I can provide a SHA here.

Reply:
Sorry, but we still need a full hash here as well! Should point to the same commit, just with a SHA rather than a tag! 🙂

Reply:
Solved.